cli/tests/testdata/main-values.yaml

```diff
@@ -31,13 +32,20 @@ timescaledb-single:
       size: 150Gi
     wal:
       size: 20Gi
+  patroni:
+    bootstrap:
+      dcs:
+        postgresql:
+          parameters:
+            max_connections: 300
 
 # Values for configuring the deployment of the Promscale Connector
 # The charts README is at:
 #   https://github.com/timescale/promscale/tree/master/helm-chart
 promscale:
   enabled: true
-  image: timescale/promscale:0.3.0
+  image: vineeth97/promscale:backfill-latest
+  replicaCount: 1
```

Comment:
shouldn't this be 2?

Reply:
So deploying promscale as `2` replicas is deploying tobs in kind CI for a very long time even without resource requests so for faster CI tests running it as `1` replica.

Reply:
In tests didn't see a reason to run more Promscale instances to speed up the CI startup of pods. Promscale needs to connect to TimescaleDB. By the time TimescaleDB is up, all Promscale pods will be in crashloopbackoff. The retries to startup Promscale instances consumes some time.

As now the control of replica count is moved to tobs CLI flag. WIll be using `--enable-prometheus-ha`.  Which internally runs 3 Promscale instances alongside Prometheus.